packages/m2m-gateway/src/utils/validators/eserviceTemplateValidators.ts

```diff
@@ -0,0 +1,13 @@
+import { eserviceTemplateApi } from "pagopa-interop-api-clients";
+import { eserviceTemplateVersionNotInSuspendedState } from "../../model/errors.js";
+
+export function assertEServiceTemplateVersionIsSuspended(
+  version: eserviceTemplateApi.EServiceTemplateVersion
+): void {
+  if (
+    version.state !==
+    eserviceTemplateApi.EServiceTemplateVersionState.Values.SUSPENDED
+  ) {
+    throw eserviceTemplateVersionNotInSuspendedState(version.id);
+  }
+}
```

Comment:
Unneeded, see: https://github.com/pagopa/interop-be-monorepo/pull/2407#discussion_r2378242011

```suggestion
```